lessons/module-2/github-collaboration.md

```diff
@@ -7,21 +7,22 @@ module: 2
 
 ## Goals
 
-By the end of this lesson, students should be able to:
-
-* Give professional and actionable feedback through code reviews
-* Recognize what makes a README useful
-* File and organize issues with labels, that are closed through PRs
+__      __.__            __     ____ _____________   ____ _______________     ___________________________________________________
+/  \    /  \  |__ _____ _/  |_  |    |   \______   \ /_   /   __   \   _  \   /  __  \_____   \_____   \_____   \_____   \_____   \
+\   \/\/   /  |  \\__  \\   __\ |    |   /|     ___/  |   \____    /  /_\  \  >      <  /   __/  /   __/  /   __/  /   __/  /   __/
+\        /|   Y  \/ __ \|  |   |    |  / |    |      |   |  /    /\  \_/   \/   --   \|   |    |   |    |   |    |   |    |   |   
+ \__/\  / |___|  (____  /__|   |______/  |____|      |___| /____/  \_____  /\______  /|___|    |___|    |___|    |___|    |___|   
+      \/       \/     \/                                                 \/        \/ <___>    <___>    <___>    <___>    <___>   
 
```

Comment:
Very Cool.  The turing site really needed this direct shout out to 1908 in my opinion.